include/SDL3/SDL_test_common.h

```diff
@@ -45,6 +45,7 @@
 #define DEFAULT_WINDOW_HEIGHT 480
 #endif
 
+typedef Uint32 SDLTest_VerboseFlags;
```

Comment:
I don't think we need to bother with this for the testing code.

Reply:
Probably, but it doesn't hurt.